service/installer/helm/helm.go

```diff
@@ -226,6 +226,7 @@ func (i *HelmInstaller) fetchMetrics() error {
 }
 
 func (i *HelmInstaller) checkHelmRelease(projectList []string) {
+	helmReleaseFailure.Reset()
```

Comment:
acutal reset point.

Reply:
I think a comment about why we need to reset would help. WDYT?